src/components/shared/sidemenu/SideMenu.jsx

```diff
@@ -35,7 +35,7 @@ const SideMenu = () => {
             to={path}
             key={name}
             className="menu-item"
```

Comment:
You can do this without the style prop, style prop will add `inline style`
```
.menu-item.active{
    color: "red",
    borderBottom:"0px"
    borderBottom: "0px",
}

```
The above code will work fine without `inline` style

Reply:
ok